repository/l.json

```diff
@@ -395,6 +395,17 @@
 				}
 			]
 		},
+		{
+			"name": "LaravelTestingCompletions",
```

Comment:
Like the other package. I've found various clunkiness with spaces in things like modules and other file path difficulties so end up using dashes, underscore, or pascal, and spaces as a last resort. 

I agree the the pascal case naming is clunky and can be hard to read when there  are more than two words. The github repo name as dashes is `sublime-laravel-testing-completions` is easier to read but dashes in module names can be difficult.

It would be nice if we separate the plugin structure from the plugin name. Does Package Control allow a display name? Maybe that the display name could parse pascal case into space separated and a pascal case plugin reserves the space case and vice versa meaning they're seen as the same by Package Control and Sublime Text.

Reply:
> Does Package Control allow a display name?

Not that I know of, the package name _is_ the directory name _is_ the package name... I do kinda like that the package name is identical between the package control website, the channel, the installation path, etc. But I can see how that's sometimes also a problem. If you want to keep it as is, that's totally understandable.